hopp/simulation/technologies/dispatch/power_storage/simple_battery_dispatch.py

```diff
@@ -68,7 +69,8 @@ def initialize_parameters(self):
 
     def _set_control_mode(self):
         """Sets control mode."""
-        if isinstance(self._system_model, BatteryModel.BatteryStateful):
+        from hopp.simulation.technologies.ldes.ldes_system_model import LDES as LDESStateful #TODO is there a more elegant way to handle this type check without a circular import?
```

Comment:
Without actually resolving the circular import another way, maybe testing that it's _not_ an instance of `BatteryStateless` might make sense? Also, is there a meaningful reason that the `LDESStateful` model has this circular import but the pySAM one doesn't? Maybe that's a hint to how we could resolve the circular import.

Lastly, I'd suggest checking in with @RHammond2 if there's not a clear cut fix; they might have some good insight!

Reply:
Fixed.